python/ray/gcs_utils.py

```diff
@@ -1,48 +1,19 @@
 from ray.core.generated.gcs_pb2 import (
-    ActorCheckpointIdData,
-    ActorTableData,
-    GcsNodeInfo,
-    JobTableData,
-    JobConfig,
-    ErrorTableData,
-    ErrorType,
-    GcsEntry,
-    HeartbeatBatchTableData,
-    HeartbeatTableData,
-    ObjectTableData,
-    ProfileTableData,
-    TablePrefix,
-    TablePubsub,
-    TaskTableData,
-    ResourceMap,
-    ResourceTableData,
-    ObjectLocationInfo,
-    PubSubMessage,
-    WorkerTableData,
-)
+    ActorCheckpointIdData, ActorTableData, GcsNodeInfo, JobTableData,
```

Comment:
How about leave the format unchanged?

Reply:
Idk why it was changed actually.. My linter just changed it. Do you know how I don't do that? (I am using clang-format7.0 & flake8).

Reply:
Fixed.

Reply:
Didn't fix actually. Let me know how I can skip linting. I will just keep it otherwise!